app/javascript/packs/clock.js

```diff
@@ -0,0 +1,31 @@
+setInterval(showTime, 1000);
+
+function showTime() {
+    let time = new Date();
+    let hour = time.getHours();
+    let min = time.getMinutes();
+    let sec = time.getSeconds();
+    let am_pm = "AM";
+
+    if (hour > 12) {
+        hour -= 12;
+        am_pm = "PM";
+    }
+    if (hour == 0) {
+        hr = 12;
+        am_pm = "AM";
+    }
+
+    hour = hour < 10 ? "0" + hour : hour;
+    min = min < 10 ? "0" + min : min;
+    sec = sec < 10 ? "0" + sec : sec;
+
+    let currentTime = hour + ":" + min;
+    let meriedem = am_pm;
```

Comment:
I get `currentTime` but looks like `meriedem` and `seconds` are not really useful variables here. 🤔

Reply:
You're right, we are working on it. Thanks